### ISSUE TITLE
Mismatching between loss function code and documentation formula

### ISSUE THREAD

**Issue by bero1403** [qiskit-community/qiskit-machine-learning#821]
According to this documentation the binary labels are part of the equation of SVCLoss, https://qiskit-community.github.io/qiskit-machine-learning/stubs/qiskit_machine_learning.utils.loss_functions.SVCLoss.html#:~:text=This%20class%20provides,is%20defined%20as%3A
However in this line of code we don't have the label in the loss function anymore.
https://github.com/qiskit-community/qiskit-machine-learning/blob/253b190eba869b4d002e490323ee4139612f01c9/qiskit_machine_learning/utils/loss_functions/kernel_loss_functions.py#L125
Can you explain why?

**Comment by adekusar-drl** [comment]
In the `SVC` documentation (https://scikit-learn.org/stable/modules/generated/sklearn.svm.SVC.html) it is stated:

```Dual coefficients of the support vector in the decision function (see [Mathematical formulation](https://scikit-learn.org/stable/modules/sgd.html#sgd-mathematical-formulation)), multiplied by their targets.```

That's why it is `np.abs(dual_coefs)` and there's no need to multiply by labels in the second term of the loss.

**Comment by edoaltamura** [comment]
Thanks @adekusar-drl. Indeed, the kernel loss function here follows the definitions of `sklearn`, which uses `dual_coef_`; see https://github.com/scikit-learn/scikit-learn/blob/934df30a7f976e2fd902b39fb0ff1c026b9c230c/sklearn/svm/_base.py#L358. 

The binary labels are not explicitly written in the loss function because they are incorporated into the kernel matrix `kmatrix` which encodes the interaction between the training samples and the labels.

You can find the formulation and more info [here](https://scikit-learn.org/stable/modules/sgd.html#sgd-mathematical-formulation). Does this answer your question, @bero1403?

**Comment by edoaltamura** [comment]
Closing this issue since it was answered. Feel free to reopen it and discuss this topic further if needed.
